mvrx/src/main/kotlin/com/airbnb/mvrx/RealMvRxStateStore.kt

```diff
@@ -101,13 +101,11 @@ class RealMvRxStateStore<S : Any>(initialState: S) : MvRxStateStore<S> {
         }
 
         @Synchronized
-        fun dequeueAllSetStateBlocks(): List<(S.() -> S)>? {
+        fun dequeueSetStateBlock(): (S.() -> S)? {
             // do not allocate empty queue for no-op flushes
```

Comment:
I don't think this comment applies anymore.

instead of doing the empty check, can you just `setStateQueue.poll()`?